tools/prologs-epilogs/gpu-health

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+#
+# Check that all GPUs are healthy via DCGM and check for ECC errors
+#
+if [ $NUMGPUS -gt 0 ]; then
+    echo "Execute DCGM health check and ECC error check for GPUs"
+    GPULIST=`nvidia-smi --query-gpu=index --format=csv,noheader | tr '\n' ',' | sed 's/,$//'`
```

Comment:
Backtick is an older way of doing this. Use `$(....)` instead

https://www.shellcheck.net/wiki/SC2006